.env.sample

```
@@ -39,4 +39,7 @@ DEBUG_LOGGING = false
 #GHOSTFOLIO_SECRET = ""
 
 # Update Cash balance when automatically importing
-#GHOSTFOLIO_UPDATE_CASH=false
\ No newline at end of file
+#GHOSTFOLIO_UPDATE_CASH=false
+
+# set locale of import file if different then en
+# IMPORT_LOCALE = "nl"
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)